db/migrate/20241106150235_change_flipper_gates_value_to_text.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class ChangeFlipperGatesValueToText < ActiveRecord::Migration[6.1]
+  def up
+    # Ensure this incremental update migration is idempotent
+    return unless connection.column_exists? :flipper_gates, :value, :string
```

Comment:
Is this really needed?  What would happen if the column were not already containing a string value, other than spending a bit longer than strictly needed doing the migration?  If you do think it's needed, should it also be used in the `down` method?

Reply:
Another alternative is to mark it as non-reversible as we've done in some previous migrations (I think I've seen it here anyway)